aim/aim_store.py

```diff
@@ -317,6 +318,14 @@ def resource_to_db_type(self, resource_klass):
         return self.db_model_map.get(resource_klass)
 
     def add(self, db_obj):
+        # "object_dict" and "tree" uses LargeBinary sqlalchemy datatype.
```

Comment:
We should remove this section of comments here.

Reply:
yes.